contributing.md

```diff
@@ -188,3 +188,13 @@ Once you're configured, building the JavaScript from the command line is easy:
     grunt watch                # watch coffee/ for changes and build Chosen
 
 If you're interested, you can find the task in [Gruntfile.coffee](https://github.com/harvesthq/chosen/blob/master/Gruntfile.coffee).
+
+#### Running tests
+
+To install all development dependencies, in the project's root directory, run
```

Comment:
It is indeed. As an experienced dev, the *only* thing I'm interested in is how do I get to run the tests. For this reason, I thought a separate (and independant) section was better. I probably chose the wrong header size. Let me tweak it and split this into another PR.